cli/cli_cities_test.go

```diff
@@ -18,7 +19,7 @@ func TestCitiesList(t *testing.T) {
 
 	tests := []struct {
 		name          string
-		cities        []string
+		cities        []*pb.ServerGroup
```

Comment:
the tests will be reviewed and more added in the next release. In this PR they were just fixed to compile.